cv.md

```
@@ -0,0 +1,29 @@
+---
+layout: page
+title: CV
+---
+
+<i>revised 10/2025</i>
+
+<iframe style="width: 100%; height: 700px" class="grid-embed__iframe" srcdoc="<!DOCTYPE html>
+	<head>
+<style>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)